RecoLocalCalo/HGCalRecProducers/test/testHGCalRecoLocal_cfg.py

```diff
@@ -11,8 +11,8 @@
 process.load('FWCore.MessageService.MessageLogger_cfi')
 process.load('Configuration.EventContent.EventContent_cff')
 process.load('SimGeneral.MixingModule.mixNoPU_cfi')
-process.load('Configuration.Geometry.GeometryExtended2023LRecoReco_cff')
-process.load('Configuration.Geometry.GeometryExtended2023LReco_cff')
+process.load('Configuration.Geometry.GeometryExtended2023simReco_cff')
```

Comment:
@lgray - this the opposite change that I would have anticipated. e.g., its the local reco geometry that is promised (intended) to always run through local reco

@kpedro88